src/legacy/core_plugins/interpreter/public/functions/tagcloud.js

```diff
@@ -31,13 +31,63 @@ export const tagcloud = () => ({
     defaultMessage: 'Tagcloud visualization'
   }),
   args: {
-    visConfig: {
-      types: ['string', 'null'],
-      default: '"{}"',
+    scale: {
+      types: ['string'],
+      default: 'linear',
+    },
+    orientation: {
+      types: ['string'],
+      default: 'single',
+    },
+    minFontSize: {
+      types: ['number'],
+      default: 18,
+    },
+    maxFontSize: {
+      types: ['number'],
+      default: 72
+    },
+    showLabel: {
+      types: ['boolean'],
+      default: true,
+    },
+    metric: {
+      types: ['string', 'number'],
     },
+    bucket: {
+      types: ['string', 'number'],
```

Comment:
When should metrics and buckets be strings vs numbers?

Reply:
if its a number its column index, if its string its column name